### PR TITLE
Added readme, license files for project

### PR DESCRIPTION
**Pull request by PRrankawat**
# Description
- Added README.md in root directory & accounts folder.
- Added license in root directory.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+         notice, this list of conditions and the following disclaimer in the
+         documentation and/or other materials provided with the distribution.
+
+      3. Neither the name of Django nor the names of its contributors may be used
+         to endorse or promote products derived from this software without
+         specific prior written permission.
+
+   THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+   ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+   WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
```
Comment (Abishek05) [inline LICENSE:25]: This is Django's LICENSE for open source projects. Please remove

Comment (PRrankawat) [reply LICENSE:25]: Okay will remove

File: README.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+```bash
+$ virtualenv project-env
+$ source project-env/bin/activate
+```
+
+### Installation
+
+1. Clone the repo
+    ```sh
+    git clone https://github.com/your_username_/Project-Name.git
```
Comment (Abishek05) [inline README.md:18]: Include our project clone URL

File: LICENSE [hunk 3]
```diff
@@ -0,0 +1,204 @@
+   Copyright (C) Java RND, Inc - All Rights Reserved
+   * Unauthorized copying of this file, via any medium is strictly prohibited  
+   * Proprietary and confidential  
+   * Written by Hirecoder Backend Team, July 2022
```
Comment (Abishek05) [inline LICENSE:4]: Change date

**Review by Abishek05 — APPROVED** [review]
Needs more changes going further

**Review by Prakharsen — APPROVED** [review]
we should have requirements.txt in repo

**Review by Abishek05 — APPROVED** [review]
Some things need to be added/modified as project progresses

**Pull request merged**